tools/manage_clvm.py

```diff
@@ -344,7 +344,7 @@ def build() -> int:
         if file_fail:
             overall_fail = True
 
-    return 1 if overall_fail else 0
+    sys.exit(1 if overall_fail else 0)
 
 
 sys.exit(main(auto_envvar_prefix="CHIA_MANAGE_CLVM"))
```

Comment:
```suggestion
main(auto_envvar_prefix="CHIA_MANAGE_CLVM")
```
Do we want to remove this also?